### PR TITLE
176971824 chem sliders

### PR DESCRIPTION
**Pull request by mklewandowski**
Adds sliders and results for the pH, nitrate, turbidity, and dissolved oxygen tests in the Chem Lab portion of the notebook.  Changes include:

- add ChemTestSlider and InputResult components to display slider and the resultant slider value
- update test results output state when slider is moved
- add icons for use with turbidity slider

Can be tested here:
https://leaf-pack.concord.org/branch/chem-sliders/

sliders

### CODE REVIEW

File: src/components/notebook/chem-test-slider.scss [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+    }
+  }
+
+  .slider {
+    margin-right: 8px;
+  }
+
+  .label {
+    margin-left: 4px;
+    color: $gray-dark
```
Comment (kswenson) [inline src/components/notebook/chem-test-slider.scss:54]:
```suggestion
    color: $gray-dark;
```

File: src/components/notebook/chem-test.tsx [hunk 2]
```diff
@@ -9,23 +11,48 @@ interface IProps {
...
   const stepsComplete = testResult?.stepsComplete ?? 0;
+  const currentStep = stepsComplete > 0
+    ? chemistryTest.steps[Math.min(stepsComplete - 1, chemistryTest.steps.length - 1)]
+    : undefined;
+  const testValueIndex = testResult?.value
+    ? chemistryTest.values.findIndex((val) => val.value === testResult.value)
+    : 0;
+
+  const handleChangeSlider = (event: any, value: number) => {
+    onUpdateTestResult(chemistryTest.type, chemistryTest.steps.length, chemistryTest.values[value].value);
```
Comment (kswenson) [inline src/components/notebook/chem-test.tsx:29]: Not sure whether there's anything to be done about it, but `chemistryTest.values[value].value` isn't the most intuitive access path I've encountered. 😉

**Review by kswenson — APPROVED** [review]
👍 Looks good!

File: src/components/notebook/chem-test.tsx (hunk 2 above)
Comment (mklewandowski) [reply src/components/notebook/chem-test.tsx]: Yeah, I hate that too.  I changed it to `results`.  That's not quite true and conflicts with some other naming.  But we can refactor later if needed.

File: src/components/notebook/chem-test-slider.scss (hunk 1 above)
Comment (mklewandowski) [reply src/components/notebook/chem-test-slider.scss]: Fixed.

**Pull request merged**